db/schema.rb

```diff
@@ -140,6 +140,198 @@
     t.datetime "updated_at", null: false
   end
 
+  create_table "versioned_asset_manager_files", force: :cascade do |t|
+    t.string "file_url"
+    t.string "state", default: "absent", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "superseded_by_id"
+    t.index ["file_url"], name: "index_versioned_asset_manager_files_on_file_url", unique: true
+  end
+
+  create_table "versioned_asset_manager_image_variants", force: :cascade do |t|
+    t.bigint "image_revision_id", null: false
+    t.bigint "asset_manager_file_id", null: false
+    t.string "variant", null: false
+    t.datetime "created_at", null: false
+    t.index ["image_revision_id", "asset_manager_file_id"], name: "index_image_revision_asset_manager_variant_ids", unique: true
+    t.index ["image_revision_id", "variant"], name: "index_image_revision_asset_manager_variant_unique_variant", unique: true
+  end
+
+  create_table "versioned_content_revisions", force: :cascade do |t|
+    t.string "title"
+    t.string "base_path"
+    t.text "summary"
+    t.json "contents", default: {}, null: false
+    t.datetime "created_at"
+    t.bigint "created_by_id"
+  end
+
+  create_table "versioned_documents", force: :cascade do |t|
+    t.uuid "content_id", null: false
+    t.string "locale", null: false
+    t.string "document_type_id", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "created_by_id"
+    t.datetime "first_published_at"
+    t.index ["content_id", "locale"], name: "index_versioned_documents_on_content_id_and_locale", unique: true
+    t.index ["created_by_id"], name: "index_versioned_documents_on_created_by_id"
+  end
+
+  create_table "versioned_editions", force: :cascade do |t|
+    t.integer "number", null: false
+    t.datetime "last_edited_at", null: false
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "document_id", null: false
+    t.bigint "created_by_id"
+    t.boolean "current", default: false, null: false
+    t.boolean "live", default: false, null: false
+    t.bigint "last_edited_by_id"
+    t.bigint "status_id", null: false
+    t.bigint "revision_id", null: false
+    t.string "draft", null: false
+    t.index ["created_by_id"], name: "index_versioned_editions_on_created_by_id"
+    t.index ["current"], name: "index_versioned_editions_on_current"
```

Comment:
It can be done - I remember doing that for a while and then undoing it. I think there's a few good reasons (that are still probably rather minor) - let me try jog my memory.

Reply:
As discussed, pointing to an edition means it would not be possible to always refer to a document directly from an edition that's not live or current; you'd need an extra pointer back to the document; while this might be preferable, the extra column (vs. live/current) makes the choice between the two less obvious.